pandas/_libs/lib.pyx

```diff
@@ -188,7 +189,7 @@ def is_scalar(val: object) -> bool:
     >>> pd.api.types.is_scalar((0, 2))
     False
 
-    pandas supports PEP 3141 numbers:
+    >>> # pandas supports PEP 3141 numbers:
```

Comment:
Can we avoid making this part of the doctest?

Reply:
skipped